src/libsyntax/print/pp.rs

```diff
@@ -300,6 +300,9 @@ impl Default for BufEntry {
     }
 }
 
+const SPACES: &str =
+    "                                                                                ";
```

Comment:
Could you add a comment telling something like "look to the right"?
I was wtf'ing on this for like a minute before noticing the closing quote.

Reply:
ok

Reply:
This could probably even use `const SPACES: [u8; 64] = [b' '; 64];` instead of the confusing string, together with writing with `self.out.write(...)` similarly to what eddyb suggests.

Reply:
Good suggestion.